slack-base/src/main/java/com/hubspot/slack/client/methods/params/chat/ChatUnfurlParamsIF.java

```diff
@@ -0,0 +1,33 @@
+package com.hubspot.slack.client.methods.params.chat;
+
+import java.net.URI;
+import java.util.Map;
+import java.util.Optional;
+
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.methods.interceptor.HasChannel;
+import com.hubspot.slack.client.models.Attachment;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface ChatUnfurlParamsIF extends HasChannel {
+  @Override
+  @JsonProperty("channel")
+  String getChannelId();
+
+  String getTs();
+
+  Map<String, Attachment> getUnfurls();
```

Comment:
We need to be [be able to handle blocks in this map too](https://api.slack.com/methods/chat.unfurl)
 > URL-encoded JSON map with keys set to URLs featured in the the message, pointing to their unfurl **blocks or message attachments**.